include/GrooveExperiments.h

```diff
@@ -0,0 +1,74 @@
+#ifndef GROOVEEXPERIMENTS_H
+#define GROOVEEXPERIMENTS_H
+
+#include <QObject>
+
+#include "Groove.h"
+#include "Knob.h"
+#include "lmms_basics.h"
+#include "MidiTime.h"
+#include "Note.h"
+#include "Pattern.h"
+
+/**
+ * A groove thats new
+ */
+class GrooveExperiments : public QObject, public Groove
+{
+	Q_OBJECT
+public:
+	GrooveExperiments(QObject *parent=0 );
+
+	virtual ~GrooveExperiments();
```

Comment:
Instead of defining `~GrooveExperiments` with an empty body in the .cpp file, you can just say `virtual ~GrooveExperiments() = default` here and omit the body.